bitmovin/resources/models/encodings/drms/playready_drm_additional_information.py

```diff
@@ -0,0 +1,23 @@
+from bitmovin.utils import Serializable
+
+
+class PlayReadyDRMAdditionalInformation(Serializable):
+
+    def __init__(self, wrm_header_custom_attributes=None):
+        super().__init__()
+        self.wrm_header_custom_attributes = wrm_header_custom_attributes
+
+    @classmethod
+    def parse_from_json_object(cls, json_object):
+        wrm_header_custom_attributes = json_object.get('wrmHeaderCustomAttributes')
+
+        playready_drm_additional_information = PlayReadyDRMAdditionalInformation(
+            wrm_header_custom_attributes=wrm_header_custom_attributes)
+
+        return playready_drm_additional_information
+
+    def serialize(self):
+        serialized = super().serialize()
+        serialized['wrmHeaderCustomAttributes'] = self.wrm_header_custom_attributes
```

Comment:
do we even need this line?

Reply:
do we even need to override the serialize function from the parent in this case?

Reply:
Yep we need that, I tried it

Reply:
approved. Yes because the variable in __init__ is defined with underscores. With that (current) solution we will have 

`wrmHeaderCustomAttributes`

and 

`wrm_header_custom_attributes`

in the serialized JSON to our api

if we don't want that we need to change `self.wrm_header_custom_attributes` to `self.wrmHeaderCustomAttributes`

and then we can also remove that serialize method

Reply:
like here https://github.com/bitmovin/bitmovin-python/blob/master/bitmovin/resources/models/manifests/dash/webm_representation.py#L17